### PR TITLE
Implement the two-fer exercise

### LINKED ISSUE #757: Implement new exercise, two-fer
In #548 we discussed a new exercise which is equivalent to the current `hello-world` exercise (which conditionally provides a name, or defaults to "world") in the string "hello, %s".

The conclusion was to implement a new exercise, `two-fer`.

- Title: Two Fer
- Slug: two-fer
- Template: `"One for X, one for me."` where X is either a name, or it defaults to `"you"`.

Next steps:

- [ ] Create the exercise description and metadata in a new PR in this repository. See https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#implementing-a-completely-new-exercise for details
- [ ] Draft an issue that can be submitted to all the repositories that currently have a `hello-world` exercise, suggesting that hello-world be simplified as described in the discussion in https://github.com/exercism/x-common/issues/520
- [ ] Get feedback on the issue text
- [ ] Submit it using `blazon` (there's some documentation about how to use blazon in  https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/improving-consistency-across-tracks.md, and an open issue about making that a stand-alone and more complete document in https://github.com/exercism/docs/issues/10)
- [ ] Close this issue (don't need to wait for all the tracks to implement it, since that is optional)

### PR DESCRIPTION
**Pull request by tejasbubane**
This is meant to be right after hello-world introducing users to basic
programming constructs as well as TDD which will be used in all later
exercises.

Discussion regarding this is [here](https://github.com/exercism/x-common/issues/548).

Closes #757.

### CODE REVIEW

File: exercises/two-fer/description.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+4. Repeat from step 1.
+5. Submit your solution (`exercism submit /path/to/file`)
+
+## Instructions
+
+Submissions are encouraged to be general, within reason. Having said that, it's
+also important not to over-engineer a solution.
+
+It's important to remember that the goal is to make code as expressive and
+readable as we can. However, solutions to the hello-world exercise will not be
```
Comment (petertseng) [inline exercises/two-fer/description.md]: hello-world

Comment (tejasbubane) [inline exercises/two-fer/description.md]: I am assuming this exercise will be reviewed by other users unlike `hello-world` which was being reviewed only by rikki. So I guess I should remove this sentence altogether.

File: exercises/two-fer/description.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+`Two-fer` or `2-fer` is short for two for one. One for you and one for me.
+
+This exercise expects you to create a sentence of the form:
```
Comment (petertseng) [inline exercises/two-fer/description.md]: Will this sound repetitive when the sentence from the blurb is prepended?

Comment (Insti) [inline exercises/two-fer/description.md]: We really should stop doing that. (prepending blurb)

**Comment by behrtam** [comment]
I would remove all the TDD stuff from the exercise description. We have different `docs` for those kind of stuff.

File: exercises/two-fer/description.md (hunk 2 above)
Comment (tejasbubane) [inline exercises/two-fer/description.md]: Right, this will be repetitive, I will remove this.

**Comment by tejasbubane** [comment]
> I would remove all the TDD stuff from the exercise description. We have different docs for those kind of stuff.

They were there in `hello-world` hence I put them here. I don't mind removing them if we have docs, but then we should atleast put a link to the docs.

**Comment by tejasbubane** [comment]
Can we merge this?

**Comment by rpottsoh** [comment]
Seems this has been sitting as approved for long enough.  If any other issues come up a new issue/PR can be opened to address it.  I'm going to merge this.

**Pull request merged**